msteams-platform/tutorials/code-samples.md

```diff
@@ -3,11 +3,21 @@ title: Microsoft Teams code samples
 description: Links and descriptions of sample applications for the Microsoft Teams developer platform
 keywords: Microsoft Teams developer samples
 ---
-# Code samples for the Microsoft Teams developer platform
+# Tutorials and code samples for the Microsoft Teams developer platform
 
-Here you will find a list of code samples that demonstrate various capabilities of the Microsoft Teams development platform and how to build apps to leverage those features.
+Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.
 
-## Getting samples
+## Getting started with learning modules
+
+| Capability| Learning Module|
+|--------|-------------|
+| Tabs  — embedded web experiences  |  [Create embedded web experiences with tabs for Microsoft Teams](https://docs.microsoft.com/learn/modules/embedded-web-experiences/) |
+| Webhooks and connectors  |  [Connect web services to Microsoft Teams with webhooks and Office 365 Connectors](https://docs.microsoft.com/learn/modules/msteams-webhooks-connectors/) |
+|Messaging extensions  | [Task-oriented interactions in Microsoft Teams with messaging extensions](https://docs.microsoft.com/learn/modules/msteams-messaging-extensions/)  |
+| Task modules |  [Collect input in Microsoft Teams with Task Modules](https://docs.microsoft.com/learn/modules/msteams-task-modules/) |
+| Conversational bots  | [Create interactive conversational bots for Microsoft Teams](https://docs.microsoft.com/learn/modules/msteams-conversation-bots/)  |
+
+## Getting started with code samples
```

Comment:
Should probably add links to the Python samples in the Messaging Extensions section.

Reply:
Also add a link to the all-up bot samples page in the intro of the bot section: https://github.com/Microsoft/BotBuilder-Samples